extensions/wikia/CuratedContent/CuratedContentController.class.php

```diff
@@ -440,14 +440,24 @@ public function getData( ) {
 
 		if ( !empty( $wgWikiaCuratedContent ) && is_array( $wgWikiaCuratedContent )  ) {
 			foreach ( $wgWikiaCuratedContent as $section ) {
-				// sections
-				if ( !empty( $section['title'] ) && empty( $section['featured'] ) ) {
+				// update information about node type
+				$section['node_type'] = 'section';
+
+				// rename $section['title'] to $section['label']
+				$section['label'] = $section['title'];
+				unset( $section['title'] );
+
+				if ( !empty( $section['label'] ) && empty( $section['featured'] ) ) {
```

Comment:
I think it doesn't solve our problem.
We wanted to have distinction for sections inside "curated" block so we don't need to do multiple ifs in Mercury.
Here you add node type to all sections (featured and optional) so we still need to add this if.